NibblePlusPlusSMD/README.md

```diff
@@ -24,7 +24,7 @@ A simple PCB to test your SMD hand-soldering skills. Going down to sand grain si
 | R3         | 0402  SMD Resistor | 1        |  62 Ohm         | yellow, white package      |
 | R4         | 0201 SMD Resistor  | 1        | 330 Ohm         | green, white package       |
 | R5         | 01005 SMD Resistor | 1        |  33 Ohm         | blue, white package        |
-| R6,R7      | 0805 SMD Resistor  | 2        | 100k Ohm        | 104                        |
+| R6,R7      | 0805 SMD Resistor  | 2        | 100k Ohm        | 104 <- im Päckchen sind andere Widerstände                   |
```

Comment:
And what kind of resistory *exactly*? How were they labeled?

Reply:
they have the number 1003 in the top

Reply:
So what about ... | 104 / 1003 | ... for the patch?